Cargo.toml

```diff
@@ -10,13 +10,14 @@ repository = "https://github.com/tafia/quick-xml"
 keywords = ["xml", "serde", "parser", "writer", "html"]
 categories = ["asynchronous", "encoding", "parsing", "parser-implementations"]
 license = "MIT"
+rust = "1.54"
```

Comment:
The key is called spelled `rust-version`, c.f. https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

Reply:
Whoops, I was using the RFC, they must have changed it https://rust-lang.github.io/rfcs/2495-min-rust-version.html